### PR TITLE
DEB-3 Add OpenHIM Mapping Mediator docs

### PR DESCRIPTION
**Pull request by de-laz**

### CODE REVIEW

File: docs/docs/how-to/configure-openhim-mapping-mediator.mdx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+The previous example will set up a docker container that will run a custom Node.js script that can send the endpoint config to a running instance of the mediator. The `command` for running this container waits for the mediator to be up and running before the import script (endpoint.js) is started.
+
+Now, add the following files under a new volume subdirectory of the docker importer directory (`docker/importer/volume`). Add the following files for the volume directory:
+
+- `example-endpoint.json` - the example endpoint configuration to be imported. For more details on endpoint configs, visit [here](https://jembi.github.io/openhim-mediator-mapping/docs/gettingStarted/endpoints).
+- `endpoint.js` - a Node.js script that is capable of importing the `example-endpoint.json` contents into a Mapping Mediator instance.
+
+Next, update the contents of your docker `docker/compose.sh` script to include the importer docker-compose file. The file could look like the following when updated:
```
Comment (MattyJ007) [inline docs/docs/how-to/configure-openhim-mapping-mediator.mdx:70]: These conjunctions flow a bit better 🤷🏼‍♂️ 
```suggestion
Finally, update the contents of your docker `docker/compose.sh` script to include the importer docker-compose file. The file could look like the following when updated:
```

File: docs/docs/how-to/configure-openhim-mapping-mediator.mdx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+With all of the docker setup in place, it is now time to import the endpoint in Kubernetes.
+
+## Kubernetes
+
+For Kubernetes, create an importer directory under the `kubernetes` directory. Under the importer directory, add the following files and directories:
+
+- `example-endpoint-importer-job.yaml` - similar to the `docker-compose.config.yml` of the docker scripts, this file will create a Kubernetes [job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) that will be responsible for running the `endpoint.js` script.
+- `k8s.sh` - to configure, start and stop the Kubernetes importer jobs.
```
Comment (MattyJ007) [inline docs/docs/how-to/configure-openhim-mapping-mediator.mdx:114]: Excellent place for an oxford comma :P 
```suggestion
- `k8s.sh` - to configure, start, and stop the Kubernetes importer jobs.
```

File: docs/docs/how-to/configure-openhim-mapping-mediator.mdx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+  - name: example-endpoint-importer-configmap
+    files:
+      - ./volume/endpoint.js
+resources:
+  - ./example-endpoint-importer-job.yaml
+```
+
+This config generates a Kubernetes `configMap` for the `example-endpoint-importer` job based on the contents of the `example-endpoint-importer-job.yaml` Kubernetes file.
+
+Finally, update the `k8s.sh` script with the following contents:
```
Comment (MattyJ007) [inline docs/docs/how-to/configure-openhim-mapping-mediator.mdx:200]:
```suggestion
Finally, update the `k8s.sh` file with the following script:
```

File: docs/docs/how-to/configure-openhim-mapping-mediator.mdx [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+    kubectl apply -k $k8sImporterRootFilePath
+elif [ "$1" == "clean" ]; then
+    kubectl delete -k $k8sImporterRootFilePath
+    kubectl delete jobs --all
+else
+    echo "Valid options are: up, or clean"
+fi
+```
+
+With everything set up, you should now be able to start Instant OpenHIE with your custom package and the OpenHIM Mapping mediator should be running and successfully configured.
```
Comment (MattyJ007) [inline docs/docs/how-to/configure-openhim-mapping-mediator.mdx:217]:
```suggestion
With everything set up, you should now be able to start Instant OpenHIE with your custom package and the OpenHIM Mapping Mediator should be running and successfully configured.
```

File: docs/docs/how-to/configure-openhim-mapping-mediator.mdx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+With all of the docker setup in place, it is now time to import the endpoint in Kubernetes.
+
+## Kubernetes
+
+For Kubernetes, create an importer directory under the `kubernetes` directory. Under the importer directory, add the following files and directories:
+
+- `example-endpoint-importer-job.yaml` - similar to the `docker-compose.config.yml` of the docker scripts, this file will create a Kubernetes [job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) that will be responsible for running the `endpoint.js` script.
+- `k8s.sh` - to configure, start and stop the Kubernetes importer jobs.
+- `kustomization.yaml` - a tool for customizing Kubernetes configurations. More information [here](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/)
```
Comment (MattyJ007) [inline docs/docs/how-to/configure-openhim-mapping-mediator.mdx:115]:
```suggestion
- `kustomization.yaml` - a tool for customizing Kubernetes configurations. More kubernetes information [here](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/).
```

File: docs/docs/how-to/configure-openhim-mapping-mediator.mdx [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+  - OpenHIM
+  - OpenHIM Mapping Mediator
+description: How to configure OpenHIM Mapping Mediator
+---
+
+:::info
+The Instant OpenHIE architecture, codebase, and documentation are under active development and are subject to change. While we encourage adoption and extension of the Instant OpenHIE framework, we do not consider this ready for production use at this stage.
+:::
+
+OpenHIM mediators are HTTP services that are registered and tracked by the OpenHIM core itself. As such, a mediator is not unique in the way it is set up for use in an Instant OpenHIE package. The only requirement is that it be packaged as a Docker image.
```
Comment (MattyJ007) [inline docs/docs/how-to/configure-openhim-mapping-mediator.mdx:18]:
```suggestion
[OpenHIM mediators](http://openhim.org/docs/user-guide/mediators) are HTTP services that are registered and tracked by the OpenHIM Core. As such, a mediator is not unique in the way it is set up for use in an Instant OpenHIE package. The only requirement is that it be packaged as a Docker image.
```

File: docs/docs/how-to/configure-openhim-mapping-mediator.mdx [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+The OpenHIM Mapping Mediator, however, is a bit more sophisticated than the average OpenHIM mediator as it is generic and designed for re-use. This mediator usually requires an endpoint configuration(s) to be imported once the mediator is up and running. The endpoint config payload can be sent as a POST request to the `/endpoints` path of the mediator. For more information on the Mapping Mediator endpoints, read here.
```
Comment (MattyJ007) [inline docs/docs/how-to/configure-openhim-mapping-mediator.mdx:20]:
```suggestion
The [OpenHIM Mapping Mediator](https://jembi.github.io/openhim-mediator-mapping), however, is a bit more sophisticated than the average OpenHIM mediator as it is generic and designed for re-use. This mediator usually requires an endpoint configuration(s) to be imported once the mediator is up and running. The endpoint config payload can be sent as a POST request to the `/endpoints` path of the mediator. For more information on the Mapping Mediator endpoints, read here.
```

**Review by MattyJ007 — COMMENTED** [review]
Looking good Laz. Just a few pedantic comments ;)

**Comment by de-laz** [comment]
> Your branch name has the Jira ticket code in lower case :( :( :(

Jira is still able to pick up the ticket number even when in lower case. I think this only works for branch names and not for commit messages.

**Pull request merged**